enderio-base/src/main/java/com/enderio/base/common/util/BlockUtil.java

```diff
@@ -0,0 +1,40 @@
+package com.enderio.base.common.util;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.server.level.ServerLevel;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.world.BlockEvent;
+
+public final class BlockUtil {
+
+    /**
+     * Removes a block as if mined by the player. All events, drops etc are triggered but Item#mineBlock is not called on the provided tool.
+     * @param level the level containing the block to be removed
+     * @param player the player mining the block
+     * @param tool the tool being used
+     * @param pos the position of the block to be removed
+     * @return true if the block was removed
+     */
+    public static boolean removeBlock(Level level, Player player, ItemStack tool, BlockPos pos) {
+        BlockState state = level.getBlockState(pos);
+        BlockEvent.BreakEvent event = new BlockEvent.BreakEvent(level, pos, state, player);
+        MinecraftForge.EVENT_BUS.post(event);
+        if (event.isCanceled()) {
+            return false;
+        }
+        boolean removed = state.onDestroyedByPlayer(level, pos, player, true, level.getFluidState(pos));
+        if (removed) {
+            state.getBlock().destroy(level, pos, state);
+            state.getBlock().playerDestroy(level, player, pos, state, null, tool);
+            if(event.getExpToDrop() > 0 && level instanceof ServerLevel serverLevel) {
```

Comment:
popExperience already has a check for 0 xp, so it's not needed here.
also add a space between if and (